### ISSUE TITLE
Support fingerprint unlock

### ISSUE THREAD

**Issue by zhijiazheng123** [twinone/AppLocker#24]
Heard that Android M will support vanilla fingerprint unlock, hope we can also support this, it both has more secure and convenience to user.

**Comment by twinone** [comment]
Yes, but before using M apis I first need to know if I can still get the top packagename, because since 5.1 the app seems not to work anymore.
